error-messages/category.messages.js

```diff
@@ -4,6 +4,8 @@ const CATEGORY_IS_NOT_MAIN = 'CATEGORY_IS_NOT_MAIN';
 const WRONG_CATEGORY_DATA = 'WRONG_CATEGORY_DATA';
 const CATEGORY_DATA_NOT_VALID = 'CATEGORY_DATA_NOT_VALID';
 const IMAGES_NOT_PROVIDED = 'IMAGES_NOT_PROVIDED';
+const CATEGORY_VALIDATION_ERROR = 'INPUT NOT VALID';
```

Comment:
Please change` 'INPUT NOT VALID'` to ` 'INPUT_NOT_VALID'` due to our style guide.

Reply:
You already have it in `common.messages.js`